### PR TITLE
Extend oEmbed guide for new oEmbed providers (Instagram, Faceboook, Podigee)

### PR DESCRIPTION
**Pull request by gabrielhase**

### CODE REVIEW

File: content/guides/documents/includes/oembed/index.md [hunk 1]
```diff
@@ -190,3 +194,27 @@ module.exports = {
...
+2. In your Facebook App, navigate to "App Review" -> "Requests" and request a new review. Then select the "Oembed Read" permissions request and fill out all required data for the review. Plan in a couple of days for Meta to process your request.
+3. Once you have the permissions, you need to add the appId and secret to Livingdocs as follows:
+```
+oembed: {
+  credentials: {
+    facebook: {
+      clientId: 'your-client-id',
+      clientSecret: 'your-client-secret'
+    },
+    allowedCoreProviders: [{
```
Comment (ajwild) [inline content/guides/documents/includes/oembed/index.md:211]: This property needs moving up one level.

File: content/guides/documents/includes/oembed/index.md [hunk 2]
```diff
@@ -14,6 +14,10 @@ The oEmbed include service (`li-oembed-service`) allows users to embed content (
 - Twitter (`li-twitter`)
 - Vimeo (`li-vimeo`)
 - YouTube (`li-youtube`)
+- Instagram (`li-instagram`), ATTENTION: requires credentials
+- Facebook (`li-facebook-post`), ATTENTION: requires credentials
```
Comment (ajwild) [inline content/guides/documents/includes/oembed/index.md:18]: Can you use the added-in release thing here please.

File: content/guides/documents/includes/oembed/index.md [hunk 3]
```diff
@@ -14,6 +14,11 @@ The oEmbed include service (`li-oembed-service`) allows users to embed content (
 - Twitter (`li-twitter`)
 - Vimeo (`li-vimeo`)
 - YouTube (`li-youtube`)
+{{< added-in release-2022-07 >}}
```
Comment (ajwild) [inline content/guides/documents/includes/oembed/index.md:17]: It doesn't work there as it displays next to YouTube. You'll need to use it twice, once for Instagram and again for Facebook.

**Review by ajwild — CHANGES_REQUESTED** [review]

File: content/guides/documents/includes/oembed/index.md (hunk 3 above)
Comment (gabrielhase) [reply content/guides/documents/includes/oembed/index.md:18]: does this work? and how can you preview it?

Comment (ajwild) [reply content/guides/documents/includes/oembed/index.md:18]: I thought like this:
```js
- Instagram (`li-instagram`), ATTENTION: requires credentials {{< added-in release-2022-07 >}}
- Facebook (`li-facebook-post`), ATTENTION: requires credentials {{< added-in release-2022-07 >}}
```
or:
```js
- Instagram (`li-instagram`) {{< added-in release-2022-07 >}} ATTENTION: requires credentials
- Facebook (`li-facebook-post`) {{< added-in release-2022-07 >}} ATTENTION: requires credentials
```
Which ever looks better.

You can preview it with `npm run start` after installing [Hugo](https://gohugo.io/).

**Review by ajwild — APPROVED** [review]
You may as well backport Datawrapper and Spotify and add them to the list too before merging.

**Pull request merged**